surveys/partisanFeelingThermometerUS/partisanFeelingThermometerUS.cy.jsx

```diff
@@ -10,18 +10,23 @@ describe("PartisanFeelingThermometerUS", () => {
     cy.spy(dummy, "set").as("callback");
     cy.mount(<PartisanFeelingThermometerUS onComplete={dummy.set} />);
 
-    cy.get("input[type=range]")
-      .invoke("val", 25)
-      .trigger("mousedown", "center");
+    if (cy.get("#sq_102_ariaTitle > .sv-string-viewer").invoke("text").then(
```

Comment:
Could you add some comments to explain what this if statement is doing? Generally cypress doesn't like you to write conditions inside tests: https://docs.cypress.io/guides/core-concepts/conditional-testing. Is this just so that it keeps trying until it gets a democrat prompt?